frontend/server/src/DAO/Contests.php

```diff
@@ -634,6 +634,113 @@ final public static function getContestsParticipating(
+                        COUNT(contestants.identity_id) AS contestants,
+                        ANY_VALUE(organizer.username) AS organizer";
+
+        $sql = "
+            FROM
+                (SELECT
+                    pi.problemset_id
+                FROM
+                    Problemset_Identities pi
+                WHERE
+                    pi.identity_id = ?
+                UNION DISTINCT
+                SELECT
+                    p.problemset_id
+                FROM
+                    Groups_Identities gi
+                INNER JOIN
+                    Group_Roles gr ON gi.group_id = gr.group_id
+                INNER JOIN
+                    Problemsets p ON gr.acl_id = p.acl_id
+                WHERE
+                    gi.identity_id = ? AND gr.role_id = ?
+                UNION DISTINCT
+                SELECT
+                    p.problemset_id
+                FROM
+                    Teams t
+                INNER JOIN
+                    Teams_Group_Roles tgr ON t.team_group_id = tgr.team_group_id
+                INNER JOIN
+                    Problemsets p ON tgr.acl_id = p.acl_id
+                WHERE
+                    t.identity_id = ? AND tgr.role_id = ?
+                ) pps
+            INNER JOIN
+                Contests
+            ON
+                Contests.problemset_id = pps.problemset_id
+            INNER JOIN
+                Problemsets p
+            ON
+                p.problemset_id = Contests.problemset_id
+            LEFT JOIN
+                Problemset_Identities AS contestants
+            ON
+                Contests.problemset_id = contestants.problemset_id
+            INNER JOIN
+                ACLs AS a
+            ON
+                Contests.acl_id = a.acl_id
+            INNER JOIN
+                Identities AS organizer
+            ON
+                a.owner_id = organizer.user_id
+            WHERE
+                ($activeCondition OR
+                $futureCondition) AND
+                archived = 0
+            GROUP BY Contests.contest_id
+        ";
+        $params = [
+            // Direct participation
+            $identityId,
+            // Group participation
+            $identityId,
+            \OmegaUp\Authorization::CONTESTANT_ROLE,
+            // Team participation
+            $identityId,
+            \OmegaUp\Authorization::CONTESTANT_ROLE,
+        ];
+
+        $limits = '
+            ORDER BY
+                start_time,
+                finish_time
+            LIMIT 1;
+        ';
+
+        /** @var {admission_mode: string, alias: string, contest_id: int, contestants: int, description: string, finish_time: \OmegaUp\Timestamp, last_updated: \OmegaUp\Timestamp, organizer: string, original_finish_time: \OmegaUp\Timestamp, problemset_id: int, recommended: bool, rerun_id: int|null, score_mode: string, scoreboard_url: string, scoreboard_url_admin: string, start_time: \OmegaUp\Timestamp, title: string, window_length: int|null} */
```

Comment:
Ahora la consulta nos puede regresar solamente el registro encontrado o `null` en caso de que no encuentre nada, así que el tipo sería así:
```suggestion
        /** @var array{admission_mode: string, alias: string, contest_id: int, contestants: int, description: string, finish_time: \OmegaUp\Timestamp, last_updated: \OmegaUp\Timestamp, organizer: string, original_finish_time: \OmegaUp\Timestamp, problemset_id: int, recommended: bool, rerun_id: int|null, score_mode: string, scoreboard_url: string, scoreboard_url_admin: string, start_time: \OmegaUp\Timestamp, title: string, window_length: int|null}|null */
```